src/app/tutorials/tutorials.component.html

```diff
@@ -1,8 +1,19 @@
-<div class="container-fluid">
+<div class="Tutorials-container container-fluid">
   <fieldset>
     <legend>TutorialsComponent</legend>
-    <div>
-      Tutorials
+    <div class="row">
+      <div class="col-2">
+        <ul class="Tutorials-menu">
+          <li>
+            <a routerLink="/tutorials/input" [routerLinkActive]="'active'">@Input</a>
```

Comment:
You can use `routerLinkActive`  to bind a class name that is going to be set on the element if the current route match the `routerLink`